docs/concepts/projects/init.md

```diff
@@ -140,7 +140,7 @@ build-backend = "hatchling.build"
 The command can be executed with `uv run`:
 
 ```console
-$ uv run --directory example-pkg example-pkg
+$ uv run --directory src/example_pkg example-pkg
```

Comment:
@zanieb -- Should we remove `--directory` entirely, and just mention that these should be run from the assumed location?

Reply:
We could, or just use `cd`

Reply:
I wanted the examples to "work as written"